pod/perlguts.pod

```diff
@@ -1562,12 +1565,14 @@ uppercase letter is typically used to represent some kind of composite type
 of that composite type.  Some internals code makes use of this case
 relationship.  However, 'v' and 'V' (vec and v-string) are in no way related.
 
-The C<PERL_MAGIC_ext> and C<PERL_MAGIC_uvar> magic types are defined
-specifically for use by extensions and will not be used by perl itself.
-Extensions can use C<PERL_MAGIC_ext> magic to 'attach' private information
-to variables (typically objects).  This is especially useful because
-there is no way for normal perl code to corrupt this private information
-(unlike using extra elements of a hash object).
+The C<PERL_MAGIC_ext>, C<PERL_MAGIC_value> and C<PERL_MAGIC_uvar> magic types
+are defined specifically for use by extensions and will not be used by perl
+itself.  Extensions can use C<PERL_MAGIC_ext> or C<PERL_MAGIC_value> magic to
+'attach' private information to variables (typically objects).  This is
+especially useful because there is no way for normal perl code to corrupt this
+private information (unlike using extra elements of a hash object). Unlike
+C<PERL_MAGIC_ext> and C<PERL_MAGIC_uvar>, C<PERL_MAGIC_value> is a value magic,
+meaning that on localization the new value will not be magical.
```

Comment:
The wording of this final sentence feels a bit round-about. Maybe start with "C<PERL_MAGIC_value> is value magic (unlike C<PERL_MAGIC_ext> and C<PERL_MAGIC_uvar>), meaning that ..."

Reply:
Yeah that does sound better